Update docs 6 (#20)
cloudfieldcz/shieldoo-mesh-docs: docs/pages/2_admin/03_servers.md

>- __Automatic OS updates__ - For more information, see [below](/servers/#automatic-os-updates).
>- __Access Rights__ - For more information, see [below](/servers/#access-rights).
>- __Attached Services__ - For more information, see [below](/servers/#attached-services).

## Editing Servers
{: .d-inline-block }
[__ADMIN__](/admin_getting_started/#administration-modes){: style="color: white; text-decoration: none;"}
{: .label .label-purple }
{: .d-inline-block }
[__ADMIN \| EXPERT__](/admin_getting_started/#administration-modes){: style="color: white; text-decoration: none;"}
{: .label .label-green }

This section describes how to edit a server's configuration after it has been created.

1. Go to the __Network__ \| __Servers__ section.
2. Open the context menu of the server whose configuration you want to edit:

   {% include img.html src="../../images/Servers02.gif" alt="Editing Servers" %}

3. Use the provided options to edit the server:  
   - __Detail__ - Open the current information about the server and the installation instructions you used to install the server.
   - __Edit__ - Open the server's configuration, which you can edit.

>    - __Delete__ - Deletes the server from your Shieldoo Secure Network. No users or other servers will be able to connect to it.

Reviewer: An idea to add: "... and its configuration will be forever lost."

## Status Servers
{: .d-inline-block }
[__ADMIN__](/admin_getting_started/#administration-modes){: style="color: white; text-decoration: none;"}
{: .label .label-purple }
{: .d-inline-block }
[__ADMIN \| EXPERT__](/admin_getting_started/#administration-modes){: style="color: white; text-decoration: none;"}
{: .label .label-green }

<!-->
ADD TYPE OF CONNECTIONS: PEER-TO-PEER, RELAY, WEBSOCKET
- Diagram image
- Explanation
- Troubleshooting
<-->

The icons in the list of servers in the __Network__ \| __Servers__ section provide visual information about the current server status:
- ![](../../images/Servers03.png)

> The server was never connected to the network, and other enabled nodes (users and servers) can not connect to this server. The server may not have been installed yet.

Reviewer: ```suggestion
The server has never been connected to the network, and other enabled nodes (users and servers) cannot connect to this server. The server may not have been installed yet.
```
